README.md

```diff
@@ -55,15 +55,15 @@ Before you can use Nextflow Tower, you need to first deploy a Tower project, whi
 
    - A new Tower workspace called `<stack_name>` was created under this organization.
 
-   - Users listed under `S3ReadWriteAccessArns` were added as workspace participants with the `Maintain` role, which grants the following permissions:
+   - Users listed under `S3ReadWriteAccessArns` were added to a workspace team with the `Maintain` role, which grants the following permissions:
```

Comment:
Are we the only admins?

Reply:
The `nextflow-tower-admin` service account is the only owner on the organization, teams, and workspaces. This way, it's harder for the Tower setup to drift from what this script deploys.